test/fork/adapters/ChainlinkSourceAdapter.sol

```diff
@@ -45,26 +45,26 @@ contract ChainlinkSourceAdapterTest is CommonTest {
         uint256 targetTime = block.timestamp - 1 hours;
         (uint80 latestRound,,,,) = chainlink.latestRoundData();
         (int256 lookBackPrice, uint256 lookBackTimestamp) = sourceAdapter.tryLatestDataAt(targetTime, 10);
-        (, int256 answer, uint256 startedAt,,) = chainlink.getRoundData(latestRound - 1);
-        assertTrue(startedAt <= targetTime); // The time from the chainlink source is at least 1 hours old.
+        (, int256 answer,, uint256 updatedAt,) = chainlink.getRoundData(latestRound - 1);
```

Comment:
after writing the other tests (for redstone) I realized this is inconsistant with qwhat the adapter actually dones, so updated it.